rust/worker/compaction_service_config.yaml

```diff
@@ -0,0 +1,39 @@
+# Default configuration for compactor service
+# In the long term, every service should have an entry in this file
+# and this can become the global configuration file for Chroma
+# for now we nest it in the worker directory
+
+worker:
```

Comment:
We don't want separate config files here that duplicate the same config. I would prefer two entires in this file

compactor:

query:

and then they map to the corresponding structs with only the information needed for each service.